webapp/src/test/java/com/box/l10n/mojito/service/asset/VirtualAssetServiceTest.java

```diff
@@ -207,6 +207,54 @@ public void testGetTextunitWithDoNoTranslateFilter() throws Exception {
         assertFalse(translateTextUnits.get(i).getDoNotTranslate());
     }
 
+    @Test
+    public void testResetOfDoNoTranslateFilter() throws Exception {
+        Repository repository = repositoryService.createRepository(testIdWatcher.getEntityName("testResetOfDoNoTranslateFilter"));
+        VirtualAsset virtualAsset = new VirtualAsset();
+        virtualAsset.setRepositoryId(repository.getId());
+        virtualAsset.setPath("default");
+        virtualAsset = virtualAssetService.createOrUpdateVirtualAsset(virtualAsset);
+
+        virtualAssetService.addTextUnit(
```

Comment:
Does that test reproduce the issue if ran against the old implementation?

Reply:
I thought it failed, but checking it again it doesn't seem to! I'll taking a look at it

Reply:
Updated the UT, now it does fail without my changes.